src/dash-table/DataTable.js

```diff
@@ -537,6 +542,151 @@ export const propTypes = {
+     * The `value` refers to the syntax-based content of
+     * the tooltip. This value is required.
+     * The `delay` represents the delay in milliseconds before
+     * the tooltip is shown when hovering a cell. This overrides
+     * the table's `tooltip_delay` property.
+     * The `duration` represents the duration in milliseconds
+     * during which the tooltip is shown when hovering a cell.
+     * This overrides the table's `tooltip_duration` property.
+     */
+    column_conditional_tooltips: PropTypes.arrayOf(PropTypes.shape({
+        if: PropTypes.shape({
+            filter: PropTypes.string,
+            row_index: PropTypes.oneOfType([
+                PropTypes.number,
+                PropTypes.oneOf([
+                    'odd',
+                    'even'
+                ])
+            ]),
+            column_id: PropTypes.string
+        }).isRequired,
+        delay: PropTypes.number,
+        duration: PropTypes.number,
+        type: PropTypes.oneOf([
+            'text',
+            'markdown'
+        ]),
+        value: PropTypes.string.isRequired
+    })),
+
+    /**
+     * `tooltips` represents the tooltip shown
+     * for different columns and cells.
+     * The `property` name refers to the column ID. Each property
+     * contains a list of tooltips mapped to the source `data`
+     * row index.
+     *
+     * The `type` refers to the type of tooltip syntax used
+     * for the tooltip generation. Can either be `markdown`
+     * or `text`. Defaults to `text`.
+     * The `value` refers to the syntax-based content of
+     * the tooltip. This value is required.
+     * The `delay` represents the delay in milliseconds before
+     * the tooltip is shown when hovering a cell. This overrides
+     * the table's `tooltip_delay` property.
+     * The `duration` represents the duration in milliseconds
+     * during which the tooltip is shown when hovering a cell.
+     * This overrides the table's `tooltip_duration` property.
+     *
+     * Alternatively, the value of the property can also be
+     * a plain string. The `text` syntax will be used in
+     * that case.
+     */
+    tooltips: PropTypes.objectOf(PropTypes.arrayOf(
+        PropTypes.oneOfType([
+            PropTypes.string,
+            PropTypes.shape({
+                delay: PropTypes.number,
+                duration: PropTypes.number,
+                type: PropTypes.oneOf([
+                    'text',
+                    'markdown'
+                ]),
+                value: PropTypes.string.isRequired
+            })
+        ]))
+    ),
+
+    /**
+     * `tooltip_delay` represents the table-wide delay in milliseconds before
+     * the tooltip is shown when hovering a cell.
+     *
+     * Default to 350.
+     */
+    tooltip_delay: PropTypes.number,
+
+    /**
+     * `tooltip_duration` represents the table-wide duration in milliseconds
+     * during which the tooltip will be displayed when hovering a cell.
+     */
```

Comment:
also would be helpful to see the duration default as the delay default is included above.

Reply:
Done :)